pygeos/constructive.py

```diff
@@ -540,8 +540,8 @@ def polygonize(geometries, **kwargs):
     ignored.
 
     This function returns the polygons within a GeometryCollection.
-    Individual Polygons can be obtained using `get_geometry` to get
-    a single polygon or `get_parts` to get an array of polygons.
+    Individual Polygons can be obtained using ``get_geometry`` to get
+    a single polygon or ``get_parts`` to get an array of polygons.
```

Comment:
I think in theory we could instruct sphinx to automatically generate links when using single backticks (but we are not doing that right now, so the change here is certainly an improvement)

Reply:
https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-default_role